pmd-cataloger/src/main/java/sfdc/sfdx/scanner/messaging/EventKey.java

```diff
@@ -0,0 +1,56 @@
+package sfdc.sfdx.scanner.messaging;
+
+import sfdc.sfdx.scanner.messaging.MessageHandler;
+import sfdc.sfdx.scanner.messaging.MessageType;
+
+public enum EventKey {
+  // ANY NEW VALUES ADDED TO THIS ENUM MUST HAVE A CORRESPONDING ENTRY IN 'messages/EventKeyTemplates.json'!
```

Comment:
Change this comment to indicate that the corresponding entry's name should be based on the messageKey string, not the Enum key.

Reply:
Good call